scripts/lib/custom_networking.sh

```diff
@@ -0,0 +1,86 @@
+#This function will setup custom networking for the cluster
+function configure_custom_networking(){
+    local VPC_ID=$(aws eks describe-cluster --name "$CLUSTER_NAME" | grep -iwo 'vpc-[a-zA-z0-9]*' | xargs)
+    echo "VPC $VPC_ID"
+
+    local RTASSOC_ID=$(aws ec2 describe-route-tables --filters Name=vpc-id,Values=$VPC_ID Name=tag:"aws:cloudformation:logical-id",Values=PublicRouteTable | jq -r '.RouteTables[].RouteTableId')
+    
+    echo "RT table $RTASSOC_ID"
+    aws ec2 associate-vpc-cidr-block --vpc-id $VPC_ID --cidr-block 100.64.0.0/16
+
+    local AZ1=us-west-2a
+    local AZ2=us-west-2b
+    local AZ3=us-west-2c
+
+    CUST_SNET1=$(aws ec2 create-subnet --cidr-block 100.64.0.0/19 --vpc-id $VPC_ID --availability-zone $AZ1 | jq -r .Subnet.SubnetId)
+    CUST_SNET2=$(aws ec2 create-subnet --cidr-block 100.64.32.0/19 --vpc-id $VPC_ID --availability-zone $AZ2 | jq -r .Subnet.SubnetId)
+    CUST_SNET3=$(aws ec2 create-subnet --cidr-block 100.64.64.0/19 --vpc-id $VPC_ID --availability-zone $AZ3 | jq -r .Subnet.SubnetId)
+
+    echo "Subnets - $CUST_SNET1 $CUST_SNET2 $CUST_SNET3"
+    aws ec2 create-tags --resources $CUST_SNET1 --tags Key=kubernetes.io/cluster/$CLUSTER_NAME,Value=shared
+    aws ec2 create-tags --resources $CUST_SNET2 --tags Key=kubernetes.io/cluster/$CLUSTER_NAME,Value=shared
+    aws ec2 create-tags --resources $CUST_SNET3 --tags Key=kubernetes.io/cluster/$CLUSTER_NAME,Value=shared
+
+    aws ec2 associate-route-table --route-table-id $RTASSOC_ID --subnet-id $CUST_SNET1
+    aws ec2 associate-route-table --route-table-id $RTASSOC_ID --subnet-id $CUST_SNET2
+    aws ec2 associate-route-table --route-table-id $RTASSOC_ID --subnet-id $CUST_SNET3
```

Comment:
Do we need to clean these resources at the end of the job?

Reply:
These should get cleaned up when cluster is deleted since vpc will get deleted.